components/timeline/TimelineHome.vue

```diff
@@ -5,19 +5,22 @@ import { ref } from 'vue'
 const paginator = useMastoClient().v1.timelines.listHome({ limit: 30 })
 const stream = $(useStreaming(client => client.v1.stream.streamUser()))
 const homePaginator = ref(null)
+const publishSnackbar = ref(null)
 
 function reorderAndFilter(items: mastodon.v1.Status[]) {
   return reorderedTimeline(items, 'home')
 }
 
 function onPublish(status) {
   homePaginator.value.percolateStatus(status)
+  publishSnackbar.value.show()
 }
 </script>
 
 <template>
-  <div>
+  <div relative>
```

Comment:
Is this a `position: relative` thing or something more fancy?

Reply:
Yeah it's the UnoCSS attribute for position relative.